code-challenges.test.js

```diff
@@ -33,9 +81,43 @@ const nums1 = [3, 56, 90, -8, 0, 23, 6]
 const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
+describe ("when minMax is called", () => {
+    it("returns an array of the minimum and maximum numbers in that order", () => {
+        const actualMinMax = minMax(nums1)
+        const actualMinMax2 = minMax(nums2)
```

Comment:
This is an interesting approach, but it certainly does what it is supposed to. You can also invoke this inside the expect method which may be a little more concise.